CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## Unreleased
+
+We have switched to [CalVer](https://calver.org/)! Relay's version is always in
+line with the latest version of [Sentry](https://github.com/getsentry/sentry).
+
+**Features**:
+
+- Proxy and managed Relays now apply clock drift correction based on the
+  `sent_at` header emitted by SDKs. (#581)
```

Comment:
the issue links will not be automatically hyperlinked, you gotta do that manually 🤮

Reply:
They will, see https://github.com/getsentry/symbolic/releases.

Reply:
Not when you look at the changelog file:
https://github.com/getsentry/symbolic/blob/master/CHANGELOG.md

Reply:
btw vscode has an official extension for this now, so it will hyperlink and show hover for those things in the editor.

Reply:
> Not when you look at the changelog file

So far, we've been fine with taking that loss, but it's not a bad idea to improve on this. 👍